docker/runtimes/action-nodejs-8/Dockerfile

```diff
@@ -1,5 +1,16 @@
-FROM openwhisk/action-nodejs-v8:1.2.0
+FROM bucharestgold/centos7-s2i-nodejs:8.x
```

Comment:
Do I have your word that this will never change significantly enough to break anything? ;)

Reply:
Hey @lance, Jim has question for ya, see above ;)